yogstation/code/modules/mob/living/carbon/human/species_types/mothmen.dm

```diff
@@ -0,0 +1,26 @@
+/datum/species/moth
+	var/MothLightRange = 15 //In what radius shall moths look for lights to fly at?
+	var/MothLightProb = 2 //Don't want them doing it tooooo often..Unless an admin changes it :^) So change this back to 5 after testing
+
+/datum/species/moth/spec_life(mob/living/carbon/human/H)
+	. = ..()
+	if(H.stat != DEAD) //Suffer not a dead moth to move
+		MothLightProb = initial(MothLightProb)
```

Comment:
This means admins can't actually change it, since it just changes back immediately.